### PR TITLE
1. 去掉sortMode的默认排序mode

### PR DESCRIPTION
**Pull request by Cenck**
@io/github/iamazy/elasticsearch/dsl/sql/parser/QueryOrderByParser.java:37
2. 使得Or 或 and 语句支持右表达式 使用in语句
   @io/github/iamazy/elasticsearch/dsl/sql/parser/query/exact/BinaryQueryParser.java:138
3. 添加无参toDsl, toPrettyDsl
   @io.github.iamazy.elasticsearch.dsl.sql.model.ElasticSqlParseResult.toDsl()

### CODE REVIEW

File: elasticsearch-sql-core/src/main/java/io/github/iamazy/elasticsearch/dsl/sql/parser/query/exact/BinaryQueryParser.java [hunk 1]
```diff
@@ -266,6 +266,9 @@ private boolean binaryContextMatch(ElasticsearchParser.ExpressionContext express
         if (expressionContext instanceof ElasticsearchParser.NameExprContext) {
             return true;
         }
+        if (expressionContext instanceof ElasticsearchParser.InContext) {
```
Comment (iamazy) [inline elasticsearch-sql-core/src/main/java/io/github/iamazy/elasticsearch/dsl/sql/parser/query/exact/BinaryQueryParser.java:269]: 可以添加个测试用例吗

**Comment by Cenck** [comment]
好 


***@***.***
 
发件人： iamazy
发送时间： 2021-12-23 15:12
收件人： iamazy/elasticsearch-sql
抄送： Cenck; Author
主题： Re: [iamazy/elasticsearch-sql] 1. 去掉sortMode的默认排序mode (PR #66)
@iamazy commented on this pull request.


In elasticsearch-sql-core/src/main/java/io/github/iamazy/elasticsearch/dsl/sql/parser/query/exact/BinaryQueryParser.java:
> @@ -266,6 +266,9 @@ private boolean binaryContextMatch(ElasticsearchParser.ExpressionContext express
         if (expressionContext instanceof ElasticsearchParser.NameExprContext) {
             return true;
         }
+        if (expressionContext instanceof ElasticsearchParser.InContext) {

可以添加个测试用例吗
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

File: elasticsearch-sql-core/src/main/java/io/github/iamazy/elasticsearch/dsl/sql/parser/query/exact/BinaryQueryParser.java (hunk 1 above)
Comment (Cenck) [reply elasticsearch-sql-core/src/main/java/io/github/iamazy/elasticsearch/dsl/sql/parser/query/exact/BinaryQueryParser.java:269]: FixedHistoryTest.fixedOn20211221 添加了测试用例

Comment (iamazy) [reply elasticsearch-sql-core/src/main/java/io/github/iamazy/elasticsearch/dsl/sql/parser/query/exact/BinaryQueryParser.java:269]: lgtm

**Pull request merged**